airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/RecordDiffer.kt

```diff
@@ -297,8 +297,9 @@ class RecordDiffer(
                     }
                     // otherwise, just be a terrible person.
                     // we know these are the same type, so this is safe to do.
-                    else ->
+                    is Comparable<*> ->
```

Comment:
this continues to be the hackiest part of the record differ - object/array aren't comparable